MapboxAndroidDemo/src/main/AndroidManifest.xml

```diff
@@ -125,6 +125,13 @@
                 android:value="com.mapbox.mapboxandroiddemo.MainActivity"/>
         </activity>
         <activity
+            android:name=".examples.plugins.TrafficPluginActivity"
+            android:label="@string/activity_mas_traffic_plugin_title">
```

Comment:
Lets change this title `activity_mas_traffic_plugin_title` to `activity_plugin_traffic_title`